.github/workflows/jira-label.yml

```diff
@@ -0,0 +1,71 @@
+# **what?**
+# Mirrors GitHub Issue labels into Jira. Includes adding a new label
+# to an existing issue or removing a label as well
+#
+# Inputs:
+#   issue_key: the key used to prefix issue id in Jira (i.e. "CT" for issue CT-123)
+#   JIRA_BASE_URL: your URL to Jira (i.e. https://abc.atlassian.net)
+#   JIRA_USER_EMAIL: user email to log into Jira under
+#   JIRA_API_TOKEN: user's API token
+
+# **why?**
+# Jira issues needs to reflect the labels of their related GitHub Issues
+
+# **when?**
+# On labels being added or removed from GitHub Issues
+
+name: Jira Label Mirroring
+
+on:
+  workflow_call:
+    inputs:
+      issue_key:
+        required: true
+        type: string
```

Comment:
@leahwicz this is a change we will have to account for where we use this action when using this repo.